.github/workflows/changelog.yaml

```diff
@@ -2,18 +2,25 @@ name: ChangelogUpdated
 on:
```

Comment:
Maybe rename it to something like "fragment_creation.yaml", "new_creation.yaml" or something similar, as now it is not really a changelog action as it was before.